tests/symmetric.rs

```diff
@@ -0,0 +1,24 @@
+use std::{fs::{File}, io::Write, path::Path};
+
+use cas_lib::symmetric::{aes::CASAES256, cas_symmetric_encryption::CASAESEncryption};
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[test]
+    fn test_sym() {

```

Comment:
Can we change the name of this function to something more descriptive like aes_256 symmetric? I think it will help distinguish between an AES 128 test case that will probably be added down the road. Other than that I am going to approve these changes

Reply:
Yes, it is good idea to distinguish the test function name. I will make the change